### PR TITLE
Make CPGServer port and host configurable

### PR DESCRIPTION
**Pull request by ursachec**

### CODE REVIEW

File: console/src/main/scala/io/shiftleft/console/cpgqlserver/CPGQLServer.scala [hunk 1]
```diff
@@ -6,7 +6,15 @@ import java.util.concurrent.ConcurrentHashMap
 import io.shiftleft.console.embammonite.{EmbeddedAmmonite, QueryResult}
 import ujson.Obj
 
-class CPGQLServer(ammonite: EmbeddedAmmonite) extends cask.MainRoutes {
+class CPGQLServer(ammonite: EmbeddedAmmonite, serverHost: String, serverPort: Int) extends cask.MainRoutes {
```
Comment (fabsx00) [inline console/src/main/scala/io/shiftleft/console/cpgqlserver/CPGQLServer.scala:9]: If you say `ammonite: EmbeddedAmmonite, val serverHost : String, val serverPort : Int)` then you do not need the two getters below.

Comment (fabsx00) [reply console/src/main/scala/io/shiftleft/console/cpgqlserver/CPGQLServer.scala:9]: Might also be a good place for default values, but up to you.

Comment (ursachec) [reply console/src/main/scala/io/shiftleft/console/cpgqlserver/CPGQLServer.scala:9]: @fabsx00 Those aren't setters, they're overwritten methods of `cask.MainRoutes`. Don't suppose there's a better shortcut for them... As to the default values, I'd rather leave them out in this case

Comment (ursachec) [reply console/src/main/scala/io/shiftleft/console/cpgqlserver/CPGQLServer.scala:9]: Actually, could be that you're right, not sure, checking...

Comment (ursachec) [reply console/src/main/scala/io/shiftleft/console/cpgqlserver/CPGQLServer.scala:9]: It seems I was correct, merging as is and cleaning up later if necessary

**Pull request merged**